helm/docs/common-configuration.md

```diff
@@ -0,0 +1,450 @@
+
+Run the following command to create a TLS secret with your SSL certificate.
+
+```console
+# Ingress
+kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+
+# Istio
+kubectl create secret tls bold-tls -n istio-system --key <key-path> --cert <certificate-path>
+```
+
+### Map multiple domains
+
+You can map mutiple domains in both Ingress Nginx and Istio like below. While mapping multiple domains you have to include the `appBaseUrl` in any of the matching host array.
+
+FOr multiple domain scenerio the `singleHost` secret will not be considered, you have to mention your secret inside `multipleHost` section.
+
+`Ingress Nginx`
+
+```console
+loadBalancer:
+  singleHost:
+    secretName: bold-tls
+
+  multipleHost:
+    hostArray:
+      - hosts: 
+          - cd1.abc.com
+          - cd2.abc.com
+        secretName: tls-secret1
+      - hosts: 
+          - cd1.xyz.com
+          - cd2.xyz.com
+        secretName: tls-secret2
+```
+
+`Istio Ingress Gateway`
+
+```console
+loadBalancer:
+  singleHost:
+    secretName: bold-tls
+
+  multipleHost:
+    hostArray:
+      - hosts: 
+          - cd1.abc.com
+          - cd2.abc.com
+        secretName: tls-secret
+```
+
+## Auto Scaling
+
+By default autoscaling is enabled in Bold BI, to disable autoscaling please set `autoscaling.enabled=false`.
+
+```console
+autoscaling:
+  enabled: true
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+  behavior:
+    stabilizationWindowSeconds: 60
+    podsValue: 1
+    podsPeriodSeconds: 60
+    percentValue: 10
+    percentPeriodSeconds: 60
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.enabled
+      </td>
+      <td>
+       By default the autoscaling will be enabled. turn this to <i>false</i> to disable the autoscaling functionality.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.targetCPUUtilizationPercentage
+      </td>
+      <td>
+       The CPU utilization is the average CPU usage of a all pods in a deployment across the last minute divided by the requested CPU of this deployment.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.targetMemoryUtilizationPercentage
+      </td>
+      <td>
+       With this metric the HPA controller will keep the average utilization of the pods in the scaling target at the value mentioned (80%). Utilization is the ratio between the current usage of resource to the requested resources of the pod.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.behavior.stabilizationWindowSeconds
+      </td>
+      <td>
+       The stabilization window is used by the autoscaling algorithm to consider the computed desired state from the past to prevent scaling. By default the value is 60 to provide a custom downscale stabilization window of 1 minute.
```

Comment:
The stabilization window is used by the autoscaling algorithm to consider the computed desired state from the past to prevent scaling. By default, the value is 60 to provide a custom downscale stabilization window of 1 minute.

Reply:
concern addressed